app/posts/find-teacher-training/2025-11-05-improving-filtering-and-sorting.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Improving how candidates can filter and sort courses
+description: Researching filter components across government and designing and testing a solution to help candidates filter courses and see results.
+date: 2025-11-05
+tags:
+  - find
+  - filters
+  - filterable find
+related:
+  items:
+    - text: Filterable Find postgraduate training courses (Find) concept
+      description: October 2022
+      href: /find-teacher-training/filterable-find-concept/
+---
+
+## Context
+
+Research in January 2025 on [improving the ability to browse and search for courses](/find-teacher-training/improving-the-ability-to-browse-and-search-for-courses-round-1-user-research/) revealed some usability issues with the filters on Find teacher training courses (Find).
+
+### 1. Seeing the ability to filter on mobile
+
+In our research, Candidates on mobile phones did not notice the ‘Show filters’ button until it was pointed out.
+
+<img alt="Screenshot of the live site on a mobile with a grey button to open a panel that displays ways to filter courses." src="live-site-filters-button.png" style="width:50%;">
+
+In March 2025, the design ops team at DfE explored [applying filters consistently in DfE](https://dfedigital.blog.gov.uk/2025/03/03/applying-filters-consistently-in-dfe/), where they found several instances of users missing the “Show filters” button when using a mobile device.
+
+They thought users may miss this grey secondary action button for a few reasons, including:
+
+- it looks inactive
+- the colour of the button is too similar to the background colour of the filters
+- the button is too far away from the results it relates to
```

Comment:
This is an interesting assertion, given your screenshot shows the 'Show filters' button is immediately adjacent to the first search result on mobile. Do you know where it was in the Design Ops team's research?

Reply:
Actually I think this feedback came from when we originally had the 'Show filters' button at the top of the page, where it was geographically further away from the first search result. We then moved it to be adjacent, but it still isn't seen by users.